Update the table name in ssdt "Debug stored procedure"

fed-dessi opened the pull request:
The code provided to create a table in the initial example (table name "Product" in the create table statement) has a different table name from the code in the last example  (table name "Products" in the select statement). 
To keep convention (it's a table that contains products) it should be the name on the table creation that changes to "Products".

fed-dessi committed:
Update the table name in ssdt "Debug stored procedure"

The code provided to create a table in the initial example has a different table name from the code in the last example (table name "Product" vs "Products")

v-dirichards commented:
@erinstellato-ms 

Can you review the proposed changes?
 
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.
 
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

erinstellato-ms commented:
#sign-off

fed-dessi committed:
Changed [dbo].[Product] to [dbo].[Products] in debug-stored-procedures.md 

Aligned every example to reflect the table name change from [dbo].[Product] to [dbo].[Products]

fed-dessi commented:
@erinstellato-ms @v-dirichards I'm terribly sorry, I went back and checked the changes and realized that I never updated the body of the Stored Procedure creation to reflect the new table name to [dbo].[Products]!

v-dirichards commented:
> @erinstellato-ms @v-dirichards I'm terribly sorry, I went back and checked the changes and realized that I never updated the body of the Stored Procedure creation to reflect the new table name to [dbo].[Products]!

@erinstellato-ms 	 - Could you review this proposed update to your article and enter another `#sign-off` in a comment if it's ready to merge?

erinstellato-ms commented:
@fed-dessi FYI, rather than changing all the DDL to create the table and the stored procedure, the SELECT statement could have been changed.

erinstellato-ms commented:
#sign-off

fed-dessi commented:
> @fed-dessi FYI, rather than changing all the DDL to create the table and the stored procedure, the SELECT statement could have been changed.

Absolutely you're correct. As I said in the first comment I thought it would've been more syntactically correct to call the table "Products" even if it's just for an example.
Thank you for the feedback I'll keep it in mind for future PRs!

v-dirichards commented:
Closing/reopening to refresh the build
